core/src/main/scala/org/broadinstitute/dsde/rawls/statistics/StatisticsService.scala

```diff
@@ -59,11 +59,9 @@ class StatisticsService(protected val userInfo: UserInfo, val dataSource: SlickD
       }
 
       DBIO.sequence(actions).flatMap { results =>
-        dataAccess.rawlsUserQuery.countUsers() flatMap { numUsers =>
-          entityStatistics.countEntitiesofTypeInNamespace(workspaceNamespace, workspaceName) map { entityStats =>
-            val allResults = results.toMap + ("currentTotalUsers" -> numUsers) + ("currentEntityStatistics" -> EntityStatistics(workspaceNamespace, workspaceName, entityStats))
```

Comment:
api change: total registered users no longer returned due to no more opendj access